nevergrad/functions/functionlib.py

```diff
@@ -63,6 +63,96 @@ def _short_repr(self) -> str:
         return "Photonics"
 
 
+class PBT_no_overfitting(ExperimentFunction):
+    """Population-Based Training, also known as Lamarckism or Meta-Optimization."""
+
+    def __init__(self, names: tp.List[str], dimensions: tp.List[int], num_workers: int):
+        for name in names:
+            if name not in corefuncs.registry:
+                available = ", ".join(sorted(corefuncs.registry))
+                raise ValueError(f'Unknown core function "{name}" in PBT. Available names are:\n-----\n{available}')
+        self._funcs = [corefuncs.registry[name] for name in names]
+        self._optima = [np.random.normal(size=d) for d in dimensions]
+        assert len(names) == len(dimensions)
+        self._dimension = len(names)
+        self._dimensions = dimensions
+        self._total_dimension = sum(dimensions)
+        parametrization = p.Array(shape=(self._dimension,)).set_name("")
+
+        # Population of checkpoints (that are optimized by the underlying optimization method)
+        # and parameters (that we do optimize).
+        self._population_checkpoints: tp.List[np.ndarray] = [np.zeros(self._total_dimension)] * num_workers
+        self._population_parameters: tp.List[np.ndarray] = [np.zeros(self._dimension)] * num_workers
+        self._population_fitness: tp.List[float] = [float("inf")] * num_workers
+        self._parameters = {x: y for x, y in locals().items() if x not in ["__class__", "self"]}
```

Comment:
what is this? this is really dangerous with so many things happening before, you should avoid using it except if there is nothing else around

Reply:
Removed.